src/product-card.css

```diff
@@ -0,0 +1,78 @@
+.product-cards {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card-img {
+  width: 160px;
+  height: 134px;
+  margin-left: 3px;
+  margin-bottom: 40px;
+}
+
+.card-name {
+  margin: 0;
```

Comment:
you can reset margins in `reset.css` so you won't repeat yourself here

Reply:
Good point, avoid doing this